Fix scroll sideways

casassg opened the pull request:
Resize title of Sponsors and Partners in mobile so we avoid scroll sideways in mobile

mvdan commented:
This diff is incredibly large. Looks like you used some kind of IDE that reindented the entire code.

Dirbaio commented:
indentas con 2 espacios
WHY

mvdan commented:
Tabs FTW ;)

No but seriously keep the current indentation, always

casassg commented:
actually I used atom :( Don't know what happened here

casassg commented:
Fixed

mvdan commented:
Why move the text? Also makes the diff larger than needed.

dasilvacontin commented:
_will @mvdan senpai approve my PR ?_
_/me le rolls :8ball:_

casassg commented:
why you don't accept me ? I moved the text because it needs to override the property when less than 600px and it was before the properties to override 

tl;dr; I moved it because css stuff made me do this

Dirbaio commented on src/styles/main.scss:
```diff
@@ -153,6 +142,20 @@ $bwhite: #FFF;
+        margin-right: 0!important;
+    }
+    .sponsors-title, .partners-title {
+      font-size: 4em;
```
bad indent :trollface:

casassg replied:
didn't see this here heh

Dirbaio commented:
:heart:
